api/app/services/faq_service.py

```
@@ -143,6 +143,87 @@ def get_all_faqs(self) -> List[FAQIdentifiedItem]:
+        return FAQListResponse(
+            faqs=paginated_faqs,
+            total_count=total_count,
+            page=page,
+            page_size=page_size,
+            total_pages=total_pages,
+        )
+
+    def _apply_filters(
+        self,
+        faqs: List[FAQIdentifiedItem],
+        search_text: str = None,
```

Ruff (0.13.1)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)